doc/Changelog.md

```diff
@@ -3,6 +3,9 @@ Changelog {#Changelog}
 
 ## Deflect 0.6
 
+### 0.6.2 (19-08-2015)
```

Comment:
done. I had hesitated to bump for such a small feature, but it is the correct thing to do :-)